src/anonymous-credentials/presentation.ts

```diff
@@ -41,8 +47,12 @@ import { LegoVerifyingKey, LegoVerifyingKeyUncompressed } from '../legosnark';
 import { SaverCiphertext } from '../saver';
 import b58 from 'bs58';
 import { SetupParamsTracker } from './setup-params-tracker';
-import { PresentationBuilder } from './presentation-builder';
 import { flattenObjectToKeyValuesList } from '../util';
+import { PSPublicKey, PSSignatureParams } from '../ps';
+import { BBSPublicKey, BBSSignatureParams } from '../bbs';
+
+type SignatureParams = BBSSignatureParams | BBSPlusSignatureParamsG2 | PSSignatureParams;
```

Comment:
These union types should be moved to a place where both the builder and this class can access these